manual/src/main/asciidoc/migrations/migrate-1.6-to-2.0.adoc

```diff
@@ -22,58 +22,58 @@ There are two main steps in preparing your migration to Apache Unomi 2.0:
 
 === Updating applications consuming Unomi
 
-Since Apache Unomi is an engine, you've probably built multiple applications consuming its APIs, you might also have built extensions directly running in Unomi. 
+Since Apache Unomi is an engine, you've probably built multiple applications consuming its APIs, you might also have built extensions directly running in Unomi.
 
-As you begin updating applications consuming Apache Unomi, it is generally a good practice to <<Enable debug mode>>. 
+As you begin updating applications consuming Apache Unomi, it is generally a good practice to <<_enabling_debug_mode,enable debug mode>>.
 Doing so will display any errors when processing events (such as JSON Schema validations), and will provide useful indications towards solving issues.
 
 ==== Data Model changes
 
-There has been changes to Unomi Data model, please make sure to review those in the << what_s_new>> section of the user manual.
+There has been changes to Unomi Data model, please make sure to review those in the <<_whats_new_in_apache_unomi_2_0,What's new in Unomi 2>> section of the user manual.
 
 ==== Create JSON schemas
 
 Once you updated your applications to align with Unomi 2 data model, the next step will be to create the necessary JSON Schemas.
 
-Any event (and more generally, any object) received through Unomi public endpoints do require a valid JSON schema. 
-Apache Unomi ships, out of the box, with all of the necessary JSON Schemas for its own operation but you will need to create schemas for any custom event you may be using.
+Any event (and more generally, any object) received through Unomi public endpoints do require a valid JSON schema.
+Apache Unomi ships, out of the box, with all of the necessary JSON Schemas for its own operation as well as all event types generated from the Apache Unomi Web Tracker but you will need to create schemas for any custom event you may be using.
 
-When creating your new schemas, reviewing debug messages in the logs (using: `log:set DEBUG org.apache.unomi.schema.impl.SchemaServiceImpl` in Karaf console), 
-will point to errors in your schemas or will help you diagnose why the events are not being accepted.
+When creating your new schemas, you can setup debug messages in the logs (using: `log:set DEBUG org.apache.unomi.schema.impl.SchemaServiceImpl` in Karaf console) that
+will point to errors in your schemas or will help you diagnose why the events are not being accepted. It is also possible to use the UNOMI_LOGS_JSONSCHEMA_LEVEL environment variable (by setting it to the `DEBUG` value) and then restarting Apache Unomi to accomplish the same thing. The second option is especially useful when using Docker containers. It is also possible to test if your events are valid with the a new API endpoint mapped at `/cxs/jsonSchema/validateEvent`.
```

Comment:
I find it a bit long to get all these in the same paragraph. You're trying to carry 3 pieces of info:
 - setup logs will help diagnose issue
 - ways to do that (3 options, karaf console, environment variable, api endpoint).

Reply:
Ok I've completely rewritten this paragraph and split it into multiple parts that should be simpler to understand.